src/main/java/bio/terra/service/dataset/flight/inheritsteward/SetInheritStewardFlight.java

```diff
@@ -61,7 +61,7 @@ public SetInheritStewardFlight(FlightMap inputParameters, Object applicationCont
       // If we are setting inherit steward to false, we want to set the flag last
       addStep(new SetInheritStewardFlagStep(datasetDao, datasetId, inheritSteward));
     }
-    addStep(new UnlockDatasetStep(datasetService, false));
+    addStep(new UnlockDatasetStep(datasetService, datasetId, false));
```

Comment:
Dataset id is not set on the dataset working map, so it needs to be passed in

Reply:
This is a downside of having two different sources of data.. We could also fix this by changing `UnlockDatasetStep` to use `FlightUtils.getContextValue()`. It turns out that `DatasetWorkingMapKeys.DATASET_ID` has the same value as `JobMapKeys.DATASET_ID` so it would "just work".

Reply:
Oh, interesting! I think I"ll keep this in this format to match the Lock step, but I'll keep that in mind in the future!